### PR TITLE
Update sonic-cli start script and resolve package dependencies

### PR DESCRIPTION
**Pull request by kwangsuk**
**- Why I did it**
As part of mgmt-framework CLI enhancement - Azure/sonic-mgmt-framework#72

**- How I did it**
update the bash script to launch Klish
update the Dockerfile.j2 to resolve dependent packages

**- How to verify it**
Incremental buster build with Azure/sonic-mgmt-framework#72

**- Which release branch to backport (provide reason below if selected)**

 201811 
 201911
 202006

**- Description for the changelog**

Update the Klish start script:
support CLI session timeout, prompt string with hostname, disallow CLI for the root user, disallow user to enter commands in click prompt after reboot command is issued in KLISH mode
add dependent libraries for Kish parser enhancement

### CODE REVIEW

**Comment by jeff-yin** [comment]
retest broadcom please

**Comment by jeff-yin** [comment]
retest vsimage please

File: dockers/docker-sonic-mgmt-framework/Dockerfile.j2 [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN [ -f /etc/rsyslog.conf ] && sed -ri "s/%syslogtag%/$docker_container_name#%s
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y g++ python-dev libxml2
+    apt-get install -y g++ python-dev libxml2 libcurl3-gnutls libpython2.7 libcjson-dev
```
[the remark this answers is not in the document]
Comment (kwangsuk) [reply dockers/docker-sonic-mgmt-framework/Dockerfile.j2:10]: Upgrade to python3.7 in sonic-cli is in progress. A PR will be opened soon after the pending PRs related to Klish parser - https://github.com/Azure/sonic-mgmt-framework/pull/72, https://github.com/Azure/sonic-mgmt-framework/pull/73 -  get merged.

**Comment by jeff-yin** [comment]
retest vsimage please

**Pull request closed without merging**